blueoil/networks/classification/quantize_example.py

```diff
@@ -0,0 +1,151 @@
+from functools import partial
+
+import tensorflow as tf
+
+from blueoil.networks.classification.base import Base
+from blueoil.layers import batch_norm, conv2d
+
+
+class SampleNetwork(Base):
+    """Sample network with simple layer."""
+
+    def __init__(
+            self,
+            *args,
+            **kwargs
+    ):
+        super().__init__(
+            *args,
+            **kwargs,
+        )
+
+        self.activation = lambda x: tf.nn.leaky_relu(x, alpha=0.1, name="leaky_relu")
+        self.before_last_activation = self.activation
```

Comment:
What is the `before_last_activation` ?

Reply:
Thank you for raise this up. 

I picked it from `darknet`. It used as a parameter to quantize activation function in the before last convolution block. 

I will fix and make use of it in this simple network.

Reply:
@iizukak Now, I keep it in the model to show how to quantize when `quantize_last_convolution=True`. I can remove it out though.

Do you think this is necessary?

Reply:
I think this is not needed for sample application.

Reply:
I removed this part out